spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe "training without cards" do
+  before :all do
+    Card.destroy_all
+    Card.create(original_text: 'дом', translated_text: 'house')
+    @card = Card.pending.first
+  end
+
+  it 'no cards' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.